src/wallet/rpc/wallet.cpp

```diff
@@ -215,12 +216,13 @@ static RPCHelpMan listwallets()
 
 static RPCHelpMan loadwallet()
 {
+    const fs::path absolutePath = GetDefaultDataDir() / fs::path("specialWallet") + fs::path::preferred_separator;
```

Comment:
Hadn't thought of the doc-generation aspect and leaks, good point.

Reply:
Just realized that in the console output in my latest test https://github.com/bitcoin/bitcoin/pull/30302#pullrequestreview-2142445372, the RPC `--user` is hardcoded to `myusername`:
```
> curl --user myusername --data-binary '{"jsonrpc": "2.0", "id": "curltest", "method": "loadwallet", "params": [/home/hodlinator/.bitcoin/specialWallet/]}'
```
So might as well use the same name if we want to go back to hardcoded platform-specific paths.

Reply:
How about just hard-coding two examples- one for Linux and other for Windows? since I generally agree with
> I am sure a user can figure out what an absolute path means without having to see it in the docs.

But there was a request [here](https://github.com/bitcoin/bitcoin/pull/30302#discussion_r1649343879) for showing the paths for different OS.

Reply:
```c++
...
+ "\nLoad wallet using absolute path (Unix):\n"
+ HelpExampleCli("loadwallet", "\"/home/myusername/specialWallet/\"")
+ HelpExampleRpc("loadwallet", "\"/home/myusername/specialWallet/\"")
+ "\nLoad wallet using absolute path (Windows):\n"
+ HelpExampleCli("loadwallet", "\"C:\\Users\\myusername\\specialWallet\\\"")
+ HelpExampleRpc("loadwallet", "\"C:\\Users\\myusername\\specialWallet\\\"")
...
```

Reply:
Maybe go more general to increase MacOS/darwin compatibility:
```
...
+ "\nLoad wallet using absolute path (Unix):\n"
+ HelpExampleCli("loadwallet", "\"/path/to/specialWallet/\"")
+ HelpExampleRpc("loadwallet", "\"/path/to/specialWallet/\"")
+ "\nLoad wallet using absolute path (Windows):\n"
+ HelpExampleCli("loadwallet", "\"C:\\Users\\myusername\\specialWallet\\\"")
+ HelpExampleRpc("loadwallet", "\"C:\\Users\\myusername\\specialWallet\\\"")
...
```

Reply:
Made this change.